GeometryService/src/ExtShieldUpstreamMaker.cc

```diff
@@ -17,7 +17,15 @@
 
 namespace mu2e {
 
-  //  namespace {
+namespace {
+   void validDoorParametersOrThrow(const double OD,const double FG){
+     if ( OD<0.0 || FG<0.0 ) {
+       throw cet::exception("GEOM")
+       << "Free Parameters, OpenDistance or FrameGap can't be negative"
+       << "\nYou supplied: OpenDistance = "<<OD<<" and FrameGap = "<<FG;
+     }
+   } // validDoorParameterOrThrow ends
+} //namespace ends
```

Comment:
Please follow the indentation pattern used by the previous authors of this file.  All 3 blocks that you touched are improperly indented.

Reply:
modified